src/ext/BalExtension/wixstdba/WixStandardBootstrapperApplication.cpp

```diff
@@ -1254,29 +1254,30 @@ class CWixStandardBootstrapperApplication : public CBalBaseBootstrapperApplicati
         hr = LocGetString(m_pWixLoc, L"#(loc.SuccessInstallHeader)", &pLocString);
         if (E_NOTFOUND == hr)
         {
-            hr = LocGetString(m_pWixLoc, L"#(loc.SuccessHeader)", &pLocString);
-            ExitOnFailure(hr, "Failed to load SuccessHeader localization string.");
-
-            hr = LocAddString(m_pWixLoc, L"SuccessInstallHeader", pLocString->wzText, pLocString->bOverridable);
-            ExitOnFailure(hr, "Failed to duplicate localization string for SuccessInstallHeader.");
-
-            hr = LocAddString(m_pWixLoc, L"SuccessRepairHeader", pLocString->wzText, pLocString->bOverridable);
-            ExitOnFailure(hr, "Failed to duplicate localization string for SuccessRepairHeader.");
+            // Duplicate strings, best-effort only.
+            if (SUCCEEDED(LocGetString(m_pWixLoc, L"#(loc.SuccessHeader)", &pLocString)))
+            {
+                hr = LocAddString(m_pWixLoc, L"SuccessInstallHeader", pLocString->wzText, pLocString->bOverridable);
```

Comment:
It succeeds, though it's essentially hidden. (The original is found first.) So the only failure here is OOM and other about-to-blow-up-anyway errors.